executor/executor.cc

```diff
@@ -850,7 +847,7 @@ void execute_one()
 	close_fds();
 #endif
 
-	if (!colliding && !collide) {
+	if (flag_extra_coverage && flag_collect_cover) {
```

Comment:
This subtly changes behavior. Previously we gave some syscalls (usb, wifi) some extra time to finish even if remove coverage is not enabled, now we don't.
Is this intentional? Why?